corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail_use_localized_description.xml

```diff
@@ -31,7 +31,7 @@
         </header>
         <template>
           <text>
-            <xpath function="report description"/>
+            <xpath function='description'/>
```

Comment:
previously the description was passed in the report fixture.  this reverts to that behavior when the localized description is used.